src/components/Button/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Loading } from '@components/Loading'
+import { HTMLAttributes } from 'react'
+import styled, { css, DefaultTheme } from 'styled-components'
+
+export interface ButtonProps extends HTMLAttributes<HTMLButtonElement> {
+  children: string
+  size: 'large' | 'medium'
+  disabled?: boolean
+  isLoading?: boolean
+  block: boolean
+}
+
+const sizeModifiers = {
+  medium: (theme: DefaultTheme) => css`
+    height: ${theme.spacings.large};
+    font-size: ${theme.fonts.sizes.large};
+  `,
+  large: (theme: DefaultTheme) => css`
+    height: ${theme.spacings.xlarge};
+    font-size: ${theme.fonts.sizes.xlarge};
+  `,
+}
+
+const Wrapper = styled.button<Omit<ButtonProps, 'children'>>`
```

Comment:
Acho que seria legal botar um min-width no botão, se não ele diminui de tamanho quando tá no estado de loading, se liga
Com min-width:

https://user-images.githubusercontent.com/20327229/186277700-9f44c0c6-2071-4d47-b3b5-e31c2753087e.mov


Atualmente sem min-width:

https://user-images.githubusercontent.com/20327229/186277646-1078a03c-7e9b-4ad2-bb9c-676d560406f1.mov

Reply:
Fix in this commit [6a2d1c](https://github.com/joaooliveiradev/conduit-frontend/pull/13/commits/6a2d1c6f2a5dfc30ef4f2f2daa339d5a01aa6d95)